<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>John Carrick Smith</author>
 <class>PdRenFiles</class>
 <widget class="QDialog" name="PdRenFiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>371</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>423</width>
    <height>371</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rename Files</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="vlMain">
     <item>
      <layout class="QVBoxLayout" name="vlDirName">
       <item>
        <widget class="QLabel" name="lblDirName">
         <property name="text">
          <string>Directory selected:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblDirNameVal">
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlFmtStr">
       <item>
        <widget class="QLabel" name="lblFormatStr">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>New Na&amp;me Format String:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="buddy">
          <cstring>txtFormatStr</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PicDokPlainTextEdit" name="txtFormatStr">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>28</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="hsFmtStr">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lblFmtDesc0">
       <property name="text">
        <string>The Following tokens have a specific meaning in the format string:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblFmt1">
       <property name="text">
        <string>%c - two digit Century
%y - two digit Year
%m - two digit Month
%d - two digit Day of month

These tokens must appear in the order above. 
Other characters will be carried into the new file name unaltered.

A sequence number will be appended controlled as below.

The files will be saved with a .jpg suffix.</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlNumLen">
       <item>
        <widget class="QLabel" name="lblNumLen">
         <property name="text">
          <string>&amp;Number of digits in Sequence No.:</string>
         </property>
         <property name="buddy">
          <cstring>cmbNumLen</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbNumLen"/>
       </item>
       <item>
        <spacer name="hsNumLen">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlStep">
       <item>
        <widget class="QLabel" name="lblNumStep">
         <property name="text">
          <string>Se&amp;quence No. step:</string>
         </property>
         <property name="buddy">
          <cstring>cmbNumStep</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbNumStep"/>
       </item>
       <item>
        <spacer name="hsNumStep">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="vsMain">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>160</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlButton">
       <item>
        <widget class="QPushButton" name="pbRenQuit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Click to abandon action.</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="hsButton">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbRenGo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Click to start process.</string>
         </property>
         <property name="text">
          <string>&amp;Go</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PicDokPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>picdokplaintextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtFormatStr</tabstop>
  <tabstop>cmbNumLen</tabstop>
  <tabstop>cmbNumStep</tabstop>
  <tabstop>pbRenGo</tabstop>
  <tabstop>pbRenQuit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbRenQuit</sender>
   <signal>clicked()</signal>
   <receiver>PdRenFiles</receiver>
   <slot>doRenQuit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbRenGo</sender>
   <signal>clicked()</signal>
   <receiver>PdRenFiles</receiver>
   <slot>doRenGo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doRenQuit()</slot>
  <slot>doRenGo()</slot>
 </slots>
</ui>
